### DISCUSSION TITLE
Reject makes hutch process stuck

### DISCUSSION THREAD

**Discussion by ahmedsaalah** [gocardless/hutch#385] [General]
After finishing process in consumer in some condition, I do reject! The Message rejected and worked as expected but the issue that hutch process stuck

**Comment by michaelklishin** [comment]
We cannot suggest much without any steps to reproduce. Look for [channel exceptions](https://www.rabbitmq.com/channels.html#error-handling) in [the server log](https://www.rabbitmq.com/logging.html), perhaps you [double acknowledge](https://www.rabbitmq.com/confirms.html#consumer-acks-double-acking) a delivery.
